### PR TITLE
mgr/balancer: add cmd to list all plans

### PR DESCRIPTION
**Pull request by yanghonggang**
Signed-off-by: Yang Honggang <yanghonggang@umcloud.com>

### CODE REVIEW

**Comment by yanghonggang** [comment]
@liewegas  please help to review

File: src/pybind/mgr/balancer/module.py [hunk 1]
```diff
@@ -271,6 +271,11 @@ class Module(MgrModule):
             "desc": "Show an optimization plan",
             "perm": "r",
         },
+        {
+            "cmd": "balancer list",
```
Comment (liewegas) [inline src/pybind/mgr/balancer/module.py]: let's to 'balancer ls' to be consistent with other ceph cli commands

**Comment by yanghonggang** [comment]
@liewegas updated

**Comment by tchaikov** [comment]
@yanghonggang could you update `doc/mgr/balancer.rst` with this newly added command?

**Comment by yanghonggang** [comment]
@tchaikov done

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2018-05-14_22:14:39-rados-wip-sage3-testing-2018-05-14-1429-distro-basic-smithi/
http://pulpito.ceph.com/sage-2018-05-16_15:59:20-rados-wip-sage3-testing-2018-05-14-1429-distro-basic-smithi/

failures are http://tracker.ceph.com/issues/24150

**Pull request merged**